README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/andela/Titan-Devs.svg?branch=ch-integrate-travisci-readme-163518672)](https://travis-ci.com/andela/Titan-Devs)
```

Comment:
@niomwungeri-fabrice, It may be better to create a build status section in the readme for this badge as said in [this tutorial](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)

Reply:
Hi @espoirMur 
Can you check it out the new version of readme?
Thanks